### PR TITLE
Retries should be on the get_url task not shell

### PR DESCRIPTION
**Pull request by domenicbove**
# Description

Small bugfix that will resolve jenkins failures. Will get merged up

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Jenkins run will test

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: roles/confluent.common/tasks/main.yml [hunk 1]
```diff
@@ -60,10 +64,6 @@
   shell: "/tmp/install.sh -b {{ confluent_cli_path | dirname }}"
   args:
     creates: "{{ confluent_cli_path }}"
-  register: cli_download_result
```
Comment (domenicbove) [inline roles/confluent.common/tasks/main.yml:63]: is removing a good idea?

Comment (domenicbove) [reply roles/confluent.common/tasks/main.yml:63]: it is not! keep it

File: roles/confluent.common/tasks/main.yml [hunk 2]
```diff
@@ -60,10 +64,6 @@
   shell: "/tmp/install.sh -b {{ confluent_cli_path | dirname }}"
   args:
     creates: "{{ confluent_cli_path }}"
```
Comment (domenicbove) [inline roles/confluent.common/tasks/main.yml:66]:
```suggestion
    creates: "{{ confluent_cli_path }}"
    register: cli_download_result
  until: cli_download_result is success
  retries: 5
  delay: 5
```

File: roles/confluent.common/tasks/main.yml [hunk 3]
```diff
@@ -60,10 +64,6 @@
...
-  register: cli_download_result
-  until: cli_download_result is success
-  retries: 5
-  delay: 5
   when:
```
Comment (domenicbove) [inline roles/confluent.common/tasks/main.yml:67]:
```suggestion
  register: cli_download_result
  until: cli_download_result is success
  retries: 5
  delay: 5
  when:
```

**Review by lokesh-shekar — APPROVED** [review]
LGTM. Thanks @domenicbove

**Pull request merged**